.github/workflows/zjit-macos.yml

```diff
@@ -38,6 +38,12 @@ jobs:
             specopts: '-T --zjit-call-threshold=1'
             test_all_opts: '--seed=46450'
 
+          - test_task: 'check'
+            configure: '--enable-zjit=dev'
```

Comment:
Could you put `configure` after the lines of `run_opts` and `specopts` for this job and the above job? I want to see `--zjit-disable-hir-opt` earlier in the job name to understand the difference between them easily.